src/vector/jitsi/index.ts

```diff
@@ -323,6 +324,11 @@ function joinConference() { // event handler bound in HTML
             SHOW_WATERMARK_FOR_GUESTS: false,
             MAIN_TOOLBAR_BUTTONS: [],
             VIDEO_LAYOUT_FIT: "height",
+            disableAEC: SettingsStore.getValue('Audio.disableAEC'),
+            disableNS: SettingsStore.getValue('Audio.disableNS'),
+            disableAP: SettingsStore.getValue('Audio.disableAP'),
+            disableAGC: SettingsStore.getValue('Audio.disableAGC'),
+            disableHPF: SettingsStore.getValue('Audio.disableHPF'),
```

Comment:
These don't match the definitions in the react-sdk, and we really shouldn't be using the SettingsStore in the Jitsi widget.

We'd instead probably want to use the widget API to send settings information down into the widget.

Reply:
Or, alternatively, these could be included in the URL fragment of the Jitsi widget, like the `isVideoChannel` parameter

Reply:
For audio settings like this I'd feel uncomfortable about having them so close to the URL processing, so would still recommend using the widget API as a cheap-ish way to do it.

I think we already have an MSC for generic config being sent to widgets anyhow.

Reply:
Oh god, I completely forgot about them when I renamed them for consistency. I am very sorry about that. 
And I will have a look into the API, the Settingstore was the first one that crossed my mind, but I will use something better if there is something.